### PR TITLE
Help Center: Fix arrow direction on RTL

### LINKED ISSUE #6259: Help Center: RTL arrow in wrong direction
Per @spacedmonkey during bug bash

Should be opposite this: 
Screen Shot 2021-02-08 at 11 17 21 AM

### PR DESCRIPTION
**Pull request by maxyinger**
## Context
Found in Bug Bash 1.4.0

## Summary
Fixes arrow direction of buttons in FTUE when in RTL mode

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
Arrows in FTUE should point in proper directions now for RTL

## Testing Instructions
-> Enable Quick Tips
-> Visit editor
-> Open Help center
-> See that arrows in main menu tips are facing the right way
-> click on a tip
-> see that `All Tips` button at bottom is facing the right way
-> switch to RTL
-> See that arrows for buttons listed above are facing the right way

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Follow Testing instructions above

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1. Follow Testing instructions above

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6259

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "web-stories-wp",
```
Comment (BrittanyIRL) [inline package-lock.json:7]: this intentional?

**Review by BrittanyIRL — APPROVED** [review]
LGTM

LTR: 
Screen Shot 2021-02-09 at 9 27 36 AM
Screen Shot 2021-02-09 at 9 27 30 AM


RTL:
Screen Shot 2021-02-09 at 9 27 59 AM
Screen Shot 2021-02-09 at 9 27 51 AM

**Review by samwhale — APPROVED** [review]
👏 👏 👏

File: package-lock.json (hunk 1 above)
Comment (maxyinger) [reply package-lock.json:7]: nope

**Pull request merged**